docs/user-guide/configuration.md

```diff
@@ -668,34 +668,33 @@ setting `use_directory_urls` to `true` or `false`.
 
 `use_directory_urls: false`
 
-This setting is needed when the documentation is hosted on systems that can't 
-access the file `X\index.html` when given the URL `X`. When set to `false`, 
-no additional `X` directory is created, and the file is simply stored as `X.html`. 
+This setting is needed when the documentation is hosted on systems that can't
+access the file `X\index.html` when given the URL `X`. When set to `false`,
```

Comment:
I suggest using a less platform-specific separator such as `/` which generally works for both Windows and Linux in Python.

```suggestion
access the file `X/index.html` when given the URL `X`. When set to `false`,
```